Source/Particles/Collision/CollisionType.cpp

```diff
@@ -156,6 +159,11 @@ void CollisionType::doCoulombCollisionsWithinTile
                     ShuffleFisherYates(
                         indices_1, cell_start_1, cell_half_1 );
 
+#if defined WARPX_DIM_RZ
+                    int ri = (i_cell - i_cell%nz) / nz;
```

Comment:
This makes more sense now, with `nz`. I might add a comment that `i_cell` varies most rapidly in z as calculated by ParticleBins.build.